apps/client/src/pages/ResultsPage.js

```diff
@@ -10,14 +12,20 @@ import Form from "../components/Form";
 import Nav from "../components/Nav";
 import QuestionAnswerTable from "../components/QuestionAnswerTable";
 import DebugDecisionTable from "../components/DebugDecisionTable";
-import { Helmet } from "react-helmet";
 
 const ResultsPage = ({ topic, checker }) => {
   const history = useHistory();
+  const sessionContext = useContext(SessionContext);
   const { slug } = topic;
 
-  const onGoToQuestion = (index) => {
-    const q = checker.rewindTo(index);
+  const onGoToQuestion = (questionIndex) => {
+    // Go to the specific question in the stack
+    const q = checker.rewindTo(questionIndex);
+    sessionContext.setSessionData({
+      questionIndex: questionIndex,
```

Comment:
you can remove the `: questionIndex`.
Doesn't our linter fix this? :-/

Reply:
Nice catch, linter did not fix this